pyigm/igm/igmsightline.py

```diff
@@ -11,12 +11,32 @@
 from linetools.isgm.abssightline import AbsSightline
 from linetools.isgm.abssystem import add_comps_from_dict
 from linetools.isgm.utils import build_systems_from_components
+from linetools import utils as ltu
 
 
 class IGMSightline(AbsSightline):
     """Class for IGM Absorption Sightline
     """
     @classmethod
+    def from_json(cls, jsonfile, coord=None, **kwargs):
+        """ Instantiate from a JSON file
+
+        Parameters
+        ----------
+        jsonfile
```

Comment:
Do the jsonfile contents adhere to some particular format?  Might be helpful to list required quantities defined within the file (RA, DEC, zem, name).